src/guide/kotlin-java.md

```diff
@@ -14,27 +14,33 @@ that you know how to build and deploy your program on the target platforms.
 The Iroha 2 JVM-compatible SDKs are as much a work-in-progress as the rest
 of this guide, and significantly more so than the Rust library.
 
-Without further ado, here's the example `build.gradle` file:
+Without further ado, here's a part of an example `build.gradle.kts` file,
+specifically, the `repositories` and `dependencies` sections:
 
 ```kotlin
-allprojects {
-    repositories {
-        maven {
-            url "https://nexus.iroha.tech/repository/maven-soramitsu/"
-            metadataSources {
-                artifact()
-            }
-        }
-        maven { url "https://jitpack.io" }
-    }
+repositories {
+    // Use Maven Central for resolving dependencies
+    mavenCentral()
+    // Use Jitpack
+    maven { url = uri("https://jitpack.io") }
 }
 
 dependencies {
-    implementation "jp.co.soramitsu.iroha2-java:client:iroha2-dev-SNAPSHOT"
-    implementation "jp.co.soramitsu.iroha2-java:block:iroha2-dev-SNAPSHOT"
-    implementation "jp.co.soramitsu.iroha2-java:model:iroha2-dev-SNAPSHOT"
-    implementation "jp.co.soramitsu.iroha2-java:test-tools:iroha2-dev-SNAPSHOT"
-    implementation "jp.co.soramitsu.iroha2-java:testcontainers:iroha2-dev-SNAPSHOT"
+    // Align versions of all Kotlin components
+    implementation(platform("org.jetbrains.kotlin:kotlin-bom"))
+    // Use the Kotlin JDK 8 standard library.
+    implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
+    // This dependency is used by the application.
```

Comment:
I'd rephrase this similarly to other comments to start with the imperative voice

Reply:
This should probably apply to the code in the SDK